src/gluonts/model/deepar/_estimator.py

```diff
@@ -489,3 +451,23 @@ def create_predictor(
             ctx=self.trainer.ctx,
             dtype=self.dtype,
         )
+
+    def get_schema(self) -> Schema:
+        fields: Dict[str, Any] = dict()
+        default_values: Dict[str, Any] = dict()
+        fields["start"] = PandasPeriodField(freq=self.freq)
+        # in the following line, we add 1 for the time dimension
+        fields["target"] = NumpyArrayField(
+            dtype=self.dtype, ndim=1 + len(self.distr_output.event_shape)
+        )
+        if self.use_feat_dynamic_real:
+            fields["feat_dynamic_real"] = NumpyArrayField(
+                dtype=self.dtype, ndim=1
+            )
+        fields["feat_static_cat"] = NumpyArrayField(dtype=self.dtype, ndim=1)
```

Comment:
This looks like we are always requiring `feat_static_cat`, even if it is set to not be used.

If I'm missing something, can you add some comments to why you do things like you do?

Last thing: I would prefer if each field is handled one after another. So this check should be together with the field definition:

```py
        if not self.use_feat_static_cat:
            default_values["feat_static_cat"] = [0.0]
```

Reply:
In `hybrid_forward`, deepAR has `feat_static_cat: Tensor` and `feat_static_real: Tensor` as input. `get_hybrid_forward_input_names` can always return what a model needs for network forward process.